docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_interactive3D.htm

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Interactive 3D</h1>
+
+<p>The interactive 3D component allows you to load a 3D model and controls the model freely.</p>
+
+<img src="../../images/wearable_interactive3D.png">
```

Comment:
please check image size.

Reply:
I used same size like as other wearable widgets.
Is it wroing?